docs/playground.html

```diff
@@ -9,6 +9,7 @@
         href="https://microsoft.github.io/monaco-editor/node_modules/monaco-editor/min/vs/editor/editor.main.css">
     <link rel="stylesheet" href="/static/shared/iframeeditor.css">
     <link rel="stylesheet" href="/static/playground/playground.css">
+    <title>MakeCode Blocks Playground</title>
```

Comment:
It is better to call it "MakeCode Playground". This also supports testing "javascript\python" as well.

Reply:
Yea that is what I was going to originally call it, but then I thought it should be consistent with the header title. Should we change both?

Reply:
Oh, I see. I will take back my comment. Looks like even though you can write "python" the focus of this tool is blocks, with even help pointing to "Defining blocks". Your change looks good.